content/renderer/BUILD.gn

```diff
@@ -0,0 +1,11 @@
+import("//build/config/features.gni")
+
+source_set("renderer") {
+  sources = [
+    "brave_render_frame_impl.cc",
+    "brave_render_frame_impl.h"
+  ]
+  public_deps = [
+    "//content/common",
+  ]
```

Comment:
`configs += [ "//content:content_implementation" ]` should be added to make `CONTENT_EXPORT` valid.